_docs/1.0.0/usage-schema.md

```diff
@@ -159,17 +155,13 @@ Data types
 
 The data type representing *String* values.
 
-| <sub><sup>Metadata keys:</sup></sub> | <sub><sup>[sourcecolumn](#sourcecolumn), [default](#default)</sup></sub> |
-|--------------------------------------|--------------------------------------------------------------------------|
-| | |
+<sup>**Metadata keys:** [sourcecolumn](#sourcecolumn), [default](#default)</sup>
```

Comment:
Why the `<sub>` removal? It places the small font text in the middle of the line.

Reply:
This should be more attached to the text above it and big enough to easily read.

In all honesty, I don't like either `sup` or `sub` but did not come with a better solution. I was thinking of defining some special class for it using `<p>` tag or something similar and still thinking about it. This is a misuse.

Reply:
Markdown is purposefully simplistic. Which comes at a price of these limitations. And the line between creativity and misuse is thin ;-)
As there is no text on the same line anyway, not that important. :-)
(For the record `<sup><sub>` is not my invention, I found it somewhere on the net)